guides/v2.3/graphql/mutations/create-payflow-pro-token.md

```diff
@@ -3,13 +3,13 @@ group: graphql
 title: createPayflowProToken mutation
 ---
 
-The `createPayflowProToken` mutation initiates a PayFlowPro transaction and receives a token. The payload must contain the redirect URLs to use when the transaction is successful, when the customer cancels the transaction, or when an error occurs.
+The `createPayflowProToken` mutation initiates a PayFlow Pro transaction and receives a token. The payload must contain the redirect URLs to use when the transaction is successful, when the customer cancels the transaction, or when an error occurs.
```

Comment:
not `Payflow Pro`? https://developer.paypal.com/docs/classic/payflow/integration-guide/

Reply:
Good catch